observability/logging/logging-6.2/log6x-configuring-lokistack-otlp-6.2.adoc

```diff
@@ -0,0 +1,145 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="log6x-configuring-lokistack-otlp-6-2"]
+= OTLP data ingestion in Loki
+include::_attributes/common-attributes.adoc[]
+:context: log6x-configuring-lokistack-otlp-6-2
+
+toc::[]
+
+Logging 6.2 enables an API endpoint using the OpenTelemetry Protocol (OTLP). As OTLP is a standardized format not specifically designed for Loki, it requires additional configuration on Loki's side to map OpenTelemetry's data format to Loki's data model. OTLP lacks concepts such as _stream labels_ or _structured metadata_. Instead, OTLP provides metadata about log entries as *attributes*, grouped into three categories:
+
+* Resource
+* Scope
+* Log
+
+This allows metadata to be set for multiple entries simultaneously or individually as needed.
+
+include::modules/log6x-configuring-lokistack-otlp-data-ingestion.adoc[leveloffset=+1]
```

Comment:
Suggestion for a follow up. In this document I believe that we can move the text in the `Prerequisites` to the paragraph right above and drop the `Prerequisites` as they do not add much value

Reply:
Added this in the Jira as a comment: https://issues.redhat.com/browse/OBSDOCS-1681?focusedId=26658126&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-26658126